Dockerfile

```diff
@@ -1,21 +1,36 @@
 FROM ubuntu:18.04
 
-ARG RELEASE_TAG
-
-ARG USERNAME=openvscode-server
-ARG USER_UID=1000
-ARG USER_GID=$USER_UID
-
 RUN apt update && \
     apt install -y git wget sudo && \
     rm -rf /var/lib/apt/lists/*
 
 WORKDIR /home/
 
+ARG RELEASE_TAG
+ARG RELEASE_ORG="gitpod-io"
```

Comment:
Was thinking about whether this is at all necessary, but I like the portability aspect, could be especially useful to users/orgs that have forks of the repo.

Reply:
Well, I was thinking about people who would like to propose changes like I did, so that they need to do less adjustments to have an environment to test.

Reply:
I'd leave it in!